superset-frontend/src/components/ObjectTags/ObjectTags.css

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
Not sure if this should be it's own file anymore. I guess so because it's pretty big, but maybe it should be .less instead?

Reply:
I think you are correct that they move moved to less